homeassistant/components/tuya/sensor.py

```diff
@@ -715,6 +715,28 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Temperature and Humidity Sensor with external probe    
+    "qxj": (
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP_CURRENT,
+            translation_key="temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.HUMIDITY_VALUE,
+            translation_key="humidity",
+            device_class=SensorDeviceClass.HUMIDITY,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP_CURRENT_EXTERNAL,
+            translation_key="temperature",
```

Comment:
This will cause two temperature sensors with the same name for this device?

That seems a bit confusing from an UX perspective.

../Frenck